libraries/botbuilder-applicationinsights/tests/test_telemetry_waterfall.py

```diff
@@ -45,7 +46,7 @@ def test_none_telemetry_client(self):
         # assert
         self.assertEqual(type(dialog.telemetry_client), NullTelemetryClient)
 
-    @skip('Pending Telemetry mock')
+
     @patch('test_telemetry_waterfall.ApplicationInsightsTelemetryClient')
```

Comment:
could you change this (and line 107) to: `@patch('botbuilder.applicationinsights.ApplicationInsightsTelemetryClient')` for the build to pass?